lib/gauge.js

```diff
@@ -1,15 +1,19 @@
 var os = require('os');
 
-module.exports = function(influxclient) {
+module.exports = function(metricsClient, type) {
```

Comment:
Is it useful to have the type here to anything apart from 'guage'? I'm just thinking that it does it differently than e.g. `lib/memory.js`, where it just sets `type.G`. Also potentially relevant: `index.js` doesn't appear to pass a second parameter, and the test does.